admin/bin/qserv-admin.py

```diff
@@ -207,65 +238,62 @@ def _parseCreateDatabase(self, tokens):
         elif l == 3:
             if tokens[1].upper() != 'LIKE':
-                raise QservAdminException(QservAdminException.BAD_CMD,
-                                    "Expected 'LIKE', found: '%s'." % tokens[1])
+                raise ExcIllegalCommand("Expected 'LIKE', found: '%s'." % tokens[1])
             dbName = tokens[0]
             dbName2 = tokens[2]
-            self._impl.createDbLike(dbName, dbName2)
+            self._css.createDbLike(dbName, dbName2)
         else:
-            raise QservAdminException(QservAdminException.BAD_CMD,
-                                "Unexpected number of arguments.")
+            raise ExcIllegalCommand("Unexpected number of arguments.")
 
     def _parseCreateTable(self, tokens):
         """
         Subparser - handles all CREATE TABLE requests.
-        Throws KvException, QservAdminException.
+        Throws css.CssError, ToolError.
         """
         l = len(tokens)
         if l == 2:
             (dbTbName, configFile) = tokens
             if '.' not in dbTbName:
-                raise QservAdminException(QservAdminException.BAD_CMD,
-                   "Invalid argument '%s', should be <dbName>.<tbName>" % dbTbName)
+                raise ExcIllegalCommand("Invalid argument '%s', should be <dbName>.<tbName>" % dbTbName)
             (dbName, tbName) = dbTbName.split('.')
             options = self._fetchOptionsFromConfigFile(configFile)
             options = self._processTbOptions(dbName, options)
-            self._impl.createTable(dbName, tbName, options)
-        elif l == 3:
-            (dbTbName, likeToken, dbTbName2) = tokens
-            if likeToken.upper() != 'LIKE':
-                raise QservAdminException(QservAdminException.BAD_CMD,
-                                    "Expected 'LIKE', found: '%s'." % tokens[2])
-            if '.' not in dbTbName:
-                raise QservAdminException(QservAdminException.BAD_CMD,
-                   "Invalid argument '%s', should be <dbName>.<tbName>" % dbTbName)
-            (dbName, tbName) = dbTbName.split('.')
-            if '.' not in dbTbName2:
-                raise QservAdminException(QservAdminException.BAD_CMD,
-                   "Invalid argument '%s', should be <dbName>.<tbName>" % dbTbName2)
-            (dbName2, tbName2) = dbTbName2.split('.')
-            self._impl.createTableLike(dbName, tbName, dbName2, tbName2,
-                                       options)
+            if options.get('match', False):
+                matchParams = css.MatchTableParams(options['dirTable1'], options['dirColName1'],
+                                                   options['dirTable2'], options['dirColName2'],
+                                                   options['flagColName'])
+                chunkLevel = 2 if int(options['subChunks']) else 1
+                self._css.createMatchTable(dbName, tbName, options['schema'], matchParams, chunkLevel)
```

Comment:
They are always chunked.

To expand on this a bit: you can of course have an unpartitioned table that records some join result, which is a "match" table in some sense. But there is no need to distinguish amongst different types of unpartitioned tables since those can be ignored by the query rewriting logic.